protocol-designer/src/load-file/migration/7_0_0.ts

```diff
@@ -240,7 +246,12 @@ export const migrateFile = (
     labwareLocationUpdate
   ).reduce((acc: LabwareLocationUpdate, labwareId: string) => {
     if (!getIsAdapter(labwareId)) {
-      acc[labwareId] = labwareLocationUpdate[labwareId]
+      const definitionId = labware[labwareId].definitionId
+      const labId =
+        labwareId === 'fixedTrash' || labwareId.includes(definitionId)
```

Comment:
nit

for consistency

```suggestion
         labwareId.includes(definitionId) || labwareId === 'fixedTrash'
```